src/components/pages/coach/CoachDashboard/index.jsx

```diff
@@ -24,11 +38,508 @@ import { ReactComponent as MorningImg } from  '../../../../assets/icons/morning.
+                    syncing: false
+                }, () => console.log("Error",response));
+            }
+        }).catch((error) => {
+            this.setState({
+                syncing: false
+            }, () => console.log("Error",error));
+        });
+    }
+
+    _loginUser = () => {
+        // Start syncing animation
+        if(!this.state.syncing){
+            this.setState({
+                syncing: true
+            });
+        }
+
+        // Login to get JWT
+        axios.post("https://charmnode.herokuapp.com/login", {
+            "username": process.env.REACT_APP_NODE_USER,
+            "password": process.env.REACT_APP_NODE_PASS
+        })
+        .then((response) => {
+            if(response.status === 200){
+                this.setState({
+                    jwt: response.data.token
+                }, () => this._fetchNodeUsers(response.data.token));
+            } else {
+                this.setState({
+                    syncing: false
+                }, () => console.log("Error",response));
+            }
+        }, (error) => {
+            this.setState({
+                syncing: false
+            }, () => console.log("Error",error));
+        });
+    }
+
+    _fetchUsersWithoutCache = () => {
+        clientNoCache.query({
+            query: GET_DATA,
+            variables: { "token": localStorage.getItem("fprint") }
+        }).then(({data}) => {
+            console.log("Fetched data directly from server.");
+            if(data.userSelf){
+                this.setState({
+                    coachusers: {
+                        userdata: data.userSelf
+                    }
+                });
+            }
+        })
+        .catch(error => {
+            this.setState({
+                syncing: false
+            }, () => console.log("Error",error));
+        })
+    }
+
+    _fetchUsersFirstTime = () => {
+        clientOnlyCache.query({
+            query: GET_DATA,
+            variables: { "token": localStorage.getItem("fprint") }
+        }).then(({data}) => {
+            console.log("Fetched data from cache.");
+            if(data.userSelf){
+                this.setState({
+                    coachusers: {
+                        userdata: data.userSelf
+                    }
+                }, () => this._fetchUsersWithoutCache());
+            }
+        })
+        .catch(error => {
+            this.setState({
+                syncing: false
+            }, () => console.log("Error",error));
+        })
+    }
+    
+    _fetchAllUsers = (customers) => {
+        clientNoCache.query({
+            query: GET_USERS_ALL,
+            variables: { "token": localStorage.getItem("fprint") }
+        }).then(({data}) => {
+            let users = data.userAll;
+            this._getDifference(users, customers);
+        })
+        .catch(error => {
+            this.setState({
+                syncing: false
+            }, () => console.log("Error",error));
```

Comment:
# please remove `console` (no-console)

In JavaScript that is designed to be executed in the browser, it's considered a best practice to avoid using methods on `console`. Such messages are considered to be for debugging purposes and therefore not suitable to ship to the client. In general, calls using `console` should be stripped before being pushed to production.

```js
console.log("Made it here.");
console.error("That shouldn't have happened.");
```

Examples of **incorrect** code for this rule:

```js
/* eslint no-console: "error" */

console.log("Log a debug level message.");
console.warn("Log a warn level message.");
console.error("Log an error level message.");
```

Reply:
Noted.
However, the console functions do not exist without reason. It is a really great, fully integrated development tool.